pkg/util/mount/mount.go

```diff
@@ -32,7 +37,19 @@ type Interface interface {
 	List() ([]MountPoint, error)
 	// IsMountPoint determines if a directory is a mountpoint.
 	IsMountPoint(file string) (bool, error)
+	SetRunner(executor ContainerExecutor)
```

Comment:
Is this really necessary to add to `mount.Interface`?  It's an implementation detail of a single mounter implementation.